server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
body-parser middleware is included with Express >=4.16 so this is a little outdated

Reply:
fixed!